ooiui/static/js/views/c2/ArraySideBarView.js

```diff
@@ -64,7 +64,6 @@ var ArraySidebarView = Backbone.View.extend({
     var self = this;
     this.collection.fetch({
```

Comment:
@ssontag55 why is the collection fetch being done in the view, this breaks the MVC approach, it would be done at the app level and when rendered